animation-done

hanisntsolo opened the pull request:
Thank you for making a pull request to ZTM's Animation Nation project. We will review your pull request as soon as we can, but keep in mind we can get very busy during Hacktoberfest.

While you're waiting, please look over this checklist. Feel free to write a description of what you did as well!

## What I did

*Optional: Edit this to tell us what you did!*

## Pull request checklist

- [x] This pull request uses **HTML and CSS** only.
- [x] All files in this pull request are in a directory named: `<github_username>-<art_name>` without the <>.
- [x] Your pull request contains only `index.html` and `styles.css`.
- [x] Your html file is named exactly: `index.html`
- [x] Your css file is named exactly: `styles.css`
- [x] Your submission includes **at least one animation**.
- [x] Your pull request **does not modify any other files** in the repository.

#### Not normally needed
- [ ] Check this if you would like to discuss an improvement besides adding animations

**Note: if your pull request makes an improvement besides adding a CSS animation you will need to discuss this with the maintainers. You can help kickstart this process by explaining what your change does and why you are making it in the "What I did" section**

## What to expect

- If your pull request meets the checklist requirements it will be approved and merged by a maintainer
- If your pull request needs some changes. A maintainer will leave a comment requesting changes
- If your pull request significantly violates the criteria in the checklist it will be closed with feedback.

LaurelineP commented:
Thanks for your contributions 🙌
Before we could merge your MR:
- [ ] please remove the icons :) 
We only need the html and css files :)

hanisntsolo commented:
> Thanks for your contributions 🙌
> 
> Before we could merge your MR:
> 
> - [ ] please remove the icons :) 
> 
> We only need the html and css files :)
> 
> 
> 
> 
> 
> 
> 
> 

Addressed the feedback!
